tests/pytests/test.py

```diff
@@ -2308,19 +2308,8 @@ def testPrefixDeletedExpansions(env):
 
     env.cmd('ft.add', 'idx', 'doc_XXX', 1, 'fields', 'txt1', 'termZZZ', 'tag1', 'tagZZZ')
 
-    # r = env.cmd('ft.search', 'idx', 'term*')
-    # print(r)
-    # r = env.cmd('ft.search', 'idx', '@tag1:{tag*}')
-    # print(r)
-
-    tmax = time.time() + 0.5  # 250ms max
-    iters = 0
-    while time.time() < tmax:
-        iters += 1
+    for i in range (10):
```

Comment:
@ashtul can you explain this change?

Reply:
@MeirShpilraien - it just tries to save a few cycles of GC.

Thinking of it, the best thing would be to add a function to common. We have many test that use `gc_forceinvoke` and we write a loop every time.

Reply:
The loop is basically because of the legacy GC which is random, I believe that in a future version we can remove the legacy GC and then the loop will not be needed.